src/EditorFeatures/Test/Diagnostics/AbstractUserDiagnosticTest.cs

```diff
@@ -293,158 +292,5 @@ protected Document GetDocumentAndAnnotatedSpan(TestWorkspace workspace, out stri
 
             Assert.True(expectedTextSpans.SetEquals(actualTextSpans));
         }
-
-        internal async Task TestWithMockedGenerateTypeDialog(
```

Comment:
Deleted code here is moved without any changes to a separate partial declaration.